content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -1006,6 +1006,63 @@ status:
   terminating: 3 # three Pods are terminating and have not yet reached the Failed phase
 ```
 
+### Success policy {#success-policy}
+
+{{< feature-state for_k8s_version="v1.29" state="alpha" >}}
+
+{{< note >}}
+You can only configure a success policy for an Indexed Job if you have the
+`JobSuccessPolicy` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
+enabled in your cluster.
+{{< /note >}}
+
+When you run an indexed Job, a success policy defined with the `spec.successPolicy` field, 
+allows you to define when a Job can be declared as succeeded based on the number of succeeded pods.
+
+In some situations, you may want to have a better control when handling Pod
+successes than the control provided by the `.spec.completins`.
+There are some examples of use cases:
+
+* To optimize costs of running workloads by avoiding unnecessary Pod running,
+  you can terminate a Job as soon as one of its Pods succeeds.
+* To care only about a leader index in determining the success or failure of a Job
+  in a batch workloads such as MPI and PyTorch etc.
+
+You can configure a success policy, in the `.spec.successPolicy` field,
+to meet the above use cases. This policy can handle Job successes based on the
+number of succeeded pods. After the Job meet success policy, the lingering Pods
+are terminated by the Job controller.
+
+When you specify the only `.spec.successPolicy.rules[*].succeededIndexes`, 
```

Comment:
```suggestion
* When you specify `.succeededIndexes` only, 
```

Reply:
Is `.succeededIndexes` top level? If not, omit the leading `.`

Reply:
The `succeededIndexes` is not top level.
So, we can omit it (`.`).

Reply:
Done.